### ISSUE TITLE
0.1.5.1 Not enough nether diamond shards from hammer mill? (moved from Subsistence Repo)

### ISSUE THREAD

**Issue by valsnoisytoys** [CyanideX/InfernalSkies#40]
Is the hammer mill supposed to give more than 1 shard per nether diamond? Because its just giving 1.

**Comment by CyanideX** [comment]
Yes it's supposed to be one shared only but of this is to challenging in terms of progression then I can increase this amount.

**Comment by valsnoisytoys** [comment]
1 feels too few to me.  My suggestion (if possible) is that it ought to be randomized range - say 1 to 3.
